.github/workflows/release.yml

```diff
@@ -48,8 +49,17 @@ jobs:
         run: pnpm install --frozen-lockfile
       - name: Build
         run: pnpm build
+
+      - name: Pre-release
+        if: ${{ inputs.distTag != 'latest' }}
+        run: pnpm run release:ci ${{inputs.releaseType}} --preRelease=${{inputs.distTag}}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Release
-        run: pnpm run release:ci --npm.tag=${{github.event.inputs.distTag}} -- ${{github.event.inputs.releaseType}}
+        if: ${{ inputs.distTag == 'latest' }}
+        run: pnpm run release:ci ${{inputs.releaseType}} --npm.tag=${{inputs.distTag}}
```

Comment:
Just checking if this is functionally the same as the line it is replacing or if we do need the ` -- ` before the releaseType.

Reply:
Explained in the comment above.